src/main/java/com/nilhcem/fakesmtp/FakeSMTP.java

```diff
@@ -53,14 +54,20 @@ public static void main(final String[] args) {
 		try {
 			ArgsHandler.INSTANCE.handleArgs(args);
 		} catch (ParseException e) {
+			System.out.println( "\n"+ e.getMessage()+ "\n" );
```

Comment:
I would suggest you to avoid the usage of **System.out**. I think it is better to use **LOGGER**.